<dec f='include/boost/property_tree/exceptions.hpp' l='48' type='void boost::property_tree::ptree_bad_data::ptree_bad_data(const std::string &amp; what, const T &amp; data)'/>
<def f='include/boost/property_tree/detail/exception_implementation.hpp' l='46' type='void boost::property_tree::ptree_bad_data::ptree_bad_data(const std::string &amp; w, const D &amp; d)'/>
<use f='include/boost/property_tree/detail/ptree_implementation.hpp' l='807' u='c' c='_ZN5boost13property_tree11basic_ptree9put_valueERKT_T0_'/>
<doc f='include/boost/property_tree/exceptions.hpp' l='43'>/// Instantiate a ptree_bad_data instance with the given message and
        /// data.
        /// @param what The message to associate with this error.
        /// @param data The value associated with this error that was the source
        ///             of the translation failure.</doc>
<doc f='include/boost/property_tree/detail/exception_implementation.hpp' l='42'>///////////////////////////////////////////////////////////////////////////
    // ptree_bad_data</doc>
